src/map/zone_entities.h

```diff
@@ -29,6 +29,9 @@ class CZoneEntities
 public:
     void HealAllMobs();
 
+    /// Finds all npcs matching the given pattern. Patterns can contain % as wildcard characters to
```

Comment:
Move long docstring to definition

Reply:
For this file, I changed to small doc to match rest of file.